cmake/EthOptions.cmake

```diff
@@ -1,7 +1,9 @@
 macro(configure_project)
 	# Default to RelWithDebInfo configuration if no configuration is explicitly specified.
-	set(CMAKE_BUILD_TYPE RelWithDebInfo CACHE STRING "Build type on single-configuration generators")
-
+	if(NOT CMAKE_BUILD_TYPE AND NOT CMAKE_CONFIGURATION_TYPES)
```

Comment:
Remove `set(CMAKE_CONFIGURATION_TYPES...)` from CMakeLists.txt.

(You may also need to clear cmake cache and configure again -- delete CMakeCache.txt in the build dir).